jsapp/js/components/transferProjects.module.scss

```diff
@@ -0,0 +1,65 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+
+.root {
+  background: colors.$kobo-white;
+  padding: sizes.$x20;
+  margin: sizes.$x24 sizes.$x40;
+  border: 1px solid colors.$kobo-gray-92;
+  border-radius: sizes.$x5;
+}
+
+.bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  button {
+    max-width: sizes.$x180;
+    height: sizes.$x38;
+  }
+}
+
+.description {
+  margin-right: sizes.$x16;
+}
+
+.copy {
+  margin-top: sizes.$x8;
+}
+
+.modalBody {
+  line-height: sizes.$x24;
+  padding: 0 sizes.$x24;
+
+  p:first-of-type {
+    margin-top: 0;
+  }
+}
+
+.warning {
+  display: flex;
+  padding: sizes.$x12;
+  margin: sizes.$x24 0 sizes.$x24 0;
+  background-color: lighten(colors.$kobo-red, 20%);
+  border-radius: sizes.$x5;
+}
+
+.warningIcon {
+  margin-right: sizes.$x8;
+}
+
+@media screen and (max-width: 768px) {
+  .bar {
+    flex-direction: column;
+    align-items: flex-start;
+
+    button {
```

Comment:
As above please add some classname to the button instead of targeting with `button` selector